setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = ldraw-to-scad
+version = 0.0.2
+author = Robert Schiele
+author_email = rschiele@gmail.com
```

Comment:
Danny, do you want to be listed as co-maintainer of the pypi packages as well or prefer to stay out here? If you want to get listed you should send me your pypi account name.

Reply:
Hello, it would be nice to be listed too. The account name is dannystaple (now I've created one).

Reply:
Okay, this gets on the list of pending changes for this draft. For now, you can see the current test version on https://test.pypi.org/project/ldraw-to-scad/. Note that this version is not working yet since I did not package the OpenSCAD library code yet, therefore a pip-installed package will fail to find that data with this test version unless when called from a checked-out source repository, which for sure a regular user would not do.

Reply:
Nice, looking forward to seeing this project there.

Reply:
Closer inspection revealed that setuptools allow only exactly one author and one maintainer. As such, I added you as the author and myself as the maintainer, which most likely is the closest match to reality.
I think this is ready now for final inspection before merging. Maybe you also want to try the package on the test server from the link above on a system that does not have access to the sources (since this is how a regular user would see it). This should install in a clean way and operate correctly. After merging this I would make a first official release to the regular pypi repository instead of the test one.